debian/run.sh

```diff
@@ -0,0 +1,13 @@
+source ../.env
+
+bash install-docker.sh
+bash lets-encrypt.sh
+# Requires copying Vault keys and root token, then pasting 3 keys back
+bash vault.sh
+# Vault root token will needed for this script and for docker-compose
+echo Paste the root token generated by Vault
+read VAULT_TOKEN
+export VAULT_TOKEN
+bash logstash.sh
+
+docker-compose -f ../docker-compose.yml up
```

Comment:
Need to add `-d` to this (to daemonize), otherwise it runs in the foreground and once you ctrl-C, it dies.

Reply:
Good catch! 19f2af1